docs/source/resources/frequently_asked_questions.ipynb

```diff
@@ -141,23 +142,23 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "es = es.normalize_entity(base_entity_id=\"transactions\",\n",
-    "                         new_entity_id=\"sessions\",\n",
+    "es = es.normalize_dataframe(base_dataframe_name=\"transactions\",\n",
+    "                         new_dataframe_name=\"sessions\",\n",
     "                         index=\"session_id\",\n",
     "                         make_time_index=\"session_start\",\n",
-    "                         additional_variables=[\"join_date\"],\n",
-    "                         copy_variables=[\"device\", \"customer_id\", \"zip_code\",\"session_start\"])"
+    "                         additional_columns=[\"join_date\"],\n",
+    "                         copy_columns=[\"device\", \"customer_id\", \"zip_code\",\"session_start\"])"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Above, we normalized the columns to create a new entity. \n",
-    "- For `additional_variables`, the following column `['join_date]` will be removed from the `products` entity, and moved to the new `device` entity. \n",
+    "Above, we normalized the columns to create a new DataFrame. \n",
     "\n",
+    "- For `additional_columns`, the following column `['join_date]` will be removed from the `products` DataFrame, and moved to the new `device` DataFrame. \n",
     "\n",
-    "- For `copy_variables`, the following columns `['device', 'customer_id', 'zip_code','session_start']` will be copied from the `products` entity to the new `device` entity. \n",
+    "- For `copy_columns`, the following columns `['device', 'customer_id', 'zip_code','session_start']` will be copied from the `products` DataFrame to the new `device` DataFrame. \n",
```

Comment:
This section seems to be talking about creating a `device` dataframe from the `products` dataframe, which is not what we're doing here. `device` should probably be replaced with `sessions`, and `products` should be replaced with `transactions`